tests/ui/test_home.py

```diff
@@ -3,15 +3,19 @@
 from pages.desktop.home import Home
 
 
+@pytest.mark.django_db
 @pytest.mark.nondestructive
-def test_there_are_ten_most_popular_extensions(base_url, selenium):
+def test_there_are_ten_most_popular_extensions(
+        base_url, selenium, live_server, initial_data):
```

Comment:
There appears to be a lot of fixtures that depend on ```base_url```. Changing the scope doesn't seem to work, but renaming it and having it return the ```live_server.url``` works.